"make" fails with permission error

Running `make` fails for me with

```
🐋  Downloaded image: raccos/opensuse:d61aef5d3cd8ec87ae0ed038082ca5979019e879cc393d2074e67f6e8441c8d5
 Error: Error met while extracting image: Could not extract rootfs: Error met while unpacking rootfs: lchown /abuild/projects/rancher/cOS/cOS-toolkit/build/rootfs325167455/tmprootfs946215145/b2d5eeeaba3a22b9b8aa97261957974a6bd65274ebd43e1d81d0a7b8b752b116.tar: operation not permitted
 Error: Error met while extracting image: Could not extract rootfs: Error met while unpacking rootfs: lchown /abuild/projects/rancher/cOS/cOS-toolkit/build/rootfs318268675/tmprootfs866460205/b2d5eeeaba3a22b9b8aa97261957974a6bd65274ebd43e1d81d0a7b8b752b116.tar: operation not permitted
 Error: Error met while generating delta: Could not generate changes from layers: Error met while unpacking src image raccos/opensuse:builder-c5772ba7c3afe4bd8688333f003532fc: Error met while unpacking rootfs: symlink ../b2d5eeeaba3a22b9b8aa97261957974a6bd65274ebd43e1d81d0a7b8b752b116.tar /var/tmp/luet/extraction756035354/src458891185/tmprootfs176377582/f190088c44f384bd5d39585e60a5e6347ffc77728da541b5abb41f7b34956a2d/layer.tar: no such file or directory
 Error: Error met while extracting image: Could not extract rootfs: Error met while unpacking rootfs: lchown /abuild/projects/rancher/cOS/cOS-toolkit/build/rootfs972745463/tmprootfs468957761/6a0451c4137ddb54788978d34ac83737ec0211cadfa526ed9c12c79a6bf34880.tar: operation not permitted
 Bailing out
make: *** [make/Makefile.build:41: build] Error 1
```

--- a/make/Makefile.build
+++ b/make/Makefile.build
@@ -37,6 +37,10 @@ PUBLISH_ARGS?=
 
 .PHONY: build
 build: luet
+ifneq ($(shell id -u), 0)
+	@echo "Please run 'make build' as root"
+	exit 1
+endif
 	@echo "PACKAGES >$(PACKAGES)<"
 	$(LUET) build $(BUILD_ARGS) \
 	--values $(ROOT_DIR)/values/$(FLAVOR).yaml \

